docs/marketplace/dev/front-end/ui-components-library/actions/actions-refresh-table.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Actions Refresh Table
+description: This article provides details about the Actions Refresh Table service in the Components Library.
+template: concept-topic-template
+---
+
+This article provides details about the Actions Refresh Table service in the Components Library.
+
+## Overview
+
+Actions Refresh Table is an Angular Service that refreshes data of the Table in current context.
+See an example below, how to use the Actions Refresh Table service.
+
+`type` - an action type.  
+`tableId` - an `id` of the table that will be refreshed via `Table Locator Service`.  
```

Comment:
Please add a link to the `Table Locator Service` doc.
Please will provide the doc if it isn't in the story list.